hack/util.sh

```diff
@@ -261,21 +262,25 @@ function wait_for_command {
 	wait=0.2
 
 	echo "[INFO] $msg"
-	expire=$(($(time_now) + $max_wait))
+	expire=$(($(time_now) + max_wait))
 	set +e
 	while [[ $(time_now) -lt $expire ]]; do
-	eval $cmd
+	# we want word splitting on the call to `eval`
+	# shellcheck disable=SC2086
+	eval ${cmd} # @miciah: can we get rid of `eval` here?
 	if [ $? -eq 0 ]; then
 		set -e
 		ENDTIME=$(date +%s)
-		echo "[INFO] Success running command: '$cmd' after $(($ENDTIME - $STARTTIME)) seconds"
+		echo "[INFO] Success running command: '$cmd' after $((ENDTIME - STARTTIME)) seconds"
 		return 0
 	fi
 	#check a failure condition where the success
 	#command may never be evaulated before timing
 	#out
 	if [[ ! -z $fail ]]; then
-		eval $fail
+		# we want word splitting on the call to `eval`
+		# shellcheck disable=SC2086
+		eval ${fail} # @miciah: can we get rid of `eval` here?
```

Comment:
Alas, we sometimes give `wait_for_command` commands that use pipes, and it seems Bash handles redirection *after* expansion, so we do need to use `eval`.  However, I don't see why we need word-splitting before `eval` runs the given command (it'll perform word-splitting when it evaluates the command).